test/fixtures/templates/good/resources/sqs/retention_period.yaml

```
@@ -0,0 +1,8 @@
+---
+AWSTemplateFormatVersion: "2010-09-09"
+Resources: 
+  MyQueue: 
+    Type: AWS::SQS::Queue
+    Properties: 
+      QueueName: "SampleQueue"
+      MessageRetentionPeriod: 5
```

```suggestion
      MessageRetentionPeriod: 60
```

Should fix this unrelated error while we're at it so the good template has no violations:
`E3034 Value has to be between 60 and 1209600 at Resources/MyQueue/Properties/MessageRetentionPeriod`